modules/persistence/src/services/pages/index.ts

```diff
@@ -43,29 +43,30 @@ const UPDATED_AST_COLL_NAME = 'updated_documents';
 // Service responsible for memoization of page level documents.
 // Any extraneous logic performed on page level documents as part of upload should be added here
 // or within subfolders of this module
-const pagesFromZip = (zip: AdmZip, githubUser?: string) => {
+const pagesFromZip = (zip: AdmZip, githubUser: string) => {
   const zipPages = zip.getEntries();
   return zipPages
     .filter((entry) => entry.entryName?.startsWith('documents/'))
     .map((entry) => {
       const document = deserialize(entry.getData());
-      document.github_username = githubUser || 'docs-builder-bot';
+      document.github_username = githubUser;
       return document;
     });
 };
 
 /**
- * Finds the page documents for a given Snooty project name + branch combination.
- * If this is the first build for the Snooty project name + branch, no documents
+ * Finds the page documents for a given Snooty project + branch + user combination.
+ * If this is the first build for the Snooty project + branch + user, no documents
  * will be found.
  *
  * @param pageIdPrefix - Includes the Snooty project name, user (docsworker-xlarge), and branch
  * @param collection - The collection to perform the find query on
  */
-const findPrevPageDocs = async (pageIdPrefix: string, collection: string) => {
+const findPrevPageDocs = async (pageIdPrefix: string, collection: string, githubUser: string) => {
   const dbSession = await db();
   const findQuery = {
     page_id: { $regex: new RegExp(`^${pageIdPrefix}/`) },
+    github_username: githubUser,
     deleted: false,
   };
   const projection = {
```

Comment:
do we want to include in the projection here for safekeeping?

Reply:
I don't think the `github_username` field is important to project here at the moment. It's not really used within the context of the returned page document

Reply:
i'm fine with leaving it out, can re insert if we need to for debugging purposes